bin/pre_deploy

```diff
@@ -1,11 +1,6 @@
 #!/usr/bin/env bash
 
 # This script is for running tasks that are 'release' rather than 'build' specific:
-# https://devcenter.heroku.com/articles/runtime-principles#build-release-run
-# It is referenced via the `release` entry in `Procfile`, and is run after the
-# buildpack compile but prior to new code being deployed:
-# https://devcenter.heroku.com/articles/release-phase
-# NB: Changes made to the filesystem will not be preserved (use `post_compile` instead).
```

Comment:
`post_compile` doesn't seem to be copied over for use by the cloudOps deployment. The file has mostly heroku related things in it but also handles the whitenoise compression with `python -m whitenoise.compress .build`. The comment about this in the file states: "is required since WhiteNoise's Django storage backend only gzips assets handled by collectstatic, and so does not affect files in the `.build/` directory since they are instead generated by Neutrino/webpack." 

@bhearsum Would you add this to the dockerfile after `RUN python manage.py collectstatic --noinput`?

Reply:
Meant to say, is this where you'd add it

Reply:
Added to dockerfile